### PR TITLE
design menu drawer, design settings screen

### PR DESCRIPTION
**Pull request by dangquanuet**

### CODE REVIEW

File: app/src/main/java/framgia/vn/weatherforecast/ui/SettingsActivity.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+    @OnClick(R.id.linear_layout_wind_speed_unit)
+    void selectWindSpeedUnits() {
+        final String[] arrayWindSpeedUnit = getResources().getStringArray(R.array.wind_speed_unit);
+        final AlertDialog.Builder builder = new AlertDialog.Builder(SettingsActivity.this);
+        builder.setTitle(R.string.wind_speed)
+                .setSingleChoiceItems(arrayWindSpeedUnit, 0, new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialog, int which) {
+                        Toast.makeText(SettingsActivity.this, "Selected " + arrayWindSpeedUnit[which], Toast.LENGTH_SHORT).show();
```
Comment (phamxuanlu) [inline app/src/main/java/framgia/vn/weatherforecast/ui/SettingsActivity.java]: hardcode String. hãy bỏ vào string.xml

Comment (dangquanuet) [inline app/src/main/java/framgia/vn/weatherforecast/ui/SettingsActivity.java]: ok anh :)

**Comment by phamxuanlu** [comment]
OK

**Comment by nvmanh** [comment]
OK!

**Pull request merged**